accounts/views.py

```diff
@@ -39,3 +40,17 @@ def register(cls, request, *args, **kwargs):
             if user:
                 return Response(serializer.data, status=status.HTTP_201_CREATED)
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    @action(detail=True, methods=["patch"], permission_classes=[IsAuthenticated])
+    def change_data(self, request, *args, **kwargs):
```

Comment:
Prepare separate serializer for changing password and create fields `old_password` and `new_password` and check it during validation. Take this as example: https://gist.github.com/gengue/a914bc1540383bfa5e1374db3df10efd

Reply:
Please also consider to implement delete account as action and also require password.